common/docker/boot_notebook.bash

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+# SEE http://redsymbol.net/articles/unofficial-bash-strict-mode/
+
+set -euo pipefail
+IFS=$'\n\t'
+INFO="INFO: [$(basename "$0")] "
+
+
+# Trust all notebooks in the notebooks folder
+echo "$INFO" "trust all notebooks in path..."
+find "${NOTEBOOK_BASE_DIR}" -name '*.ipynb' -type f | xargs -I % /bin/bash -c 'jupyter trust % || true'
+
+
+# Configure
+# Prevents notebook to open in separate tab
+mkdir -p "$HOME/.jupyter/custom"
+cat > "$HOME/.jupyter/custom/custom.js" <<EOF
+define(['base/js/namespace'], function(Jupyter){
+    Jupyter._target = '_self';
+});
+EOF
+
+#https://github.com/jupyter/notebook/issues/3130 for delete_to_trash
+#https://github.com/nteract/hydrogen/issues/922 for disable_xsrf
+cat > .jupyter_config.json <<EOF
+{
+    "FileCheckpoints": {
+        "checkpoint_dir": "/home/jovyan/._ipynb_checkpoints/"
+    },
+    "KernelSpecManager": {
+        "ensure_native_kernel": false
+    },
+    "Session": {
+        "debug": false
+    },
+    "VoilaConfiguration" : {
+        "enable_nbextensions" : true
+    },
+    "ServerApp": {
+        "quit_button": false,
+        "extra_static_paths": ["/static"],
+        "ip": "0.0.0.0",
+        "port": 8888,
+        "base_url": "",
+        "token": "",
+        "open_browser": false,
+        "disable_check_xsrf": true,
+        "webbrowser_open_new": 0,
+        "notebook_dir": "${NOTEBOOK_BASE_DIR}",
+        "root_dir": "${NOTEBOOK_BASE_DIR}",
+        "preferred_dir": "${NOTEBOOK_BASE_DIR}/workspace/"
+    }
+}
+EOF
+
+# Checking for resources limit env vars injected by osparc
+SIMCORE_NANO_CPUS_LIMIT="${SIMCORE_NANO_CPUS_LIMIT:-0}"
+if [ "${SIMCORE_NANO_CPUS_LIMIT}" -ne "0" ]
+then
+    echo "$INFO" "Found NANO_CPU limits: ${SIMCORE_NANO_CPUS_LIMIT}"
+    NANO_CPU_DIVISOR=1000000000
+    MAX_CPUS=$(("${SIMCORE_NANO_CPUS_LIMIT}" / "${NANO_CPU_DIVISOR}"))
+    # use 1 if this is 0 otherwise floor is probably fine
+    if [ "${MAX_CPUS}" -eq "0" ]
+    then
+        MAX_CPUS=1
+    fi
+    echo "$INFO" "Setting Z43_MAX_CPU_RESOURCES to " "${MAX_CPUS}"
+    export Z43_MAX_CPU_RESOURCES="${MAX_CPUS}"
+    export OMP_NUM_THREADS="${MAX_CPUS}"
+fi
+
+## set licensing information if not yet set via platform
+PORT=27000
+declare -a SERVERS=("license.speag.com" "license-1.speag.com" "10.1.0.21")
```

Comment:
same here. Licensing only makes sense for sim4life

Reply:
will remove also this